misc/aliases/minetest

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+echo -e "\033[93mminetest is a deprecated alias for luanti\033[0m"
+dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+"$dir/luanti" "$@"
```

Comment:
for completeness this would have been my complaints:
* requiring bash is probably not a good idea (not portable)
* `echo -e` is unportable
* printing color codes without regard for tty
* finding the script dir is probably unnecessary since we can assume the user has both in his PATH?
* missing `exec`